file-formats/suco/type/zif_aff_suco_v1.intf.abap

```diff
@@ -0,0 +1,44 @@
+"! AFF Type for SUCO (Authorization Default Variant)
+INTERFACE zif_aff_suco_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Leading Application</p>
+    "! Leading application
+    BEGIN OF ty_leading_application,
+      "! <p class="shorttext">Program ID</p>
+      "! Program ID
+      "! $default 'R3TR'
+      program_id   TYPE c LENGTH 4,
+      "! <p class="shorttext">Object Type</p>
+      "! Object type
+      "! $required
+      object_type  TYPE c LENGTH 4,
+      "! <p class="shorttext">Object Name</p>
+      "! Object name
+      "! $required
+      object_name  TYPE c LENGTH 40,
+      "! <p class="shorttext">Service Type</p>
+      "! Service type
+      service_type TYPE c LENGTH 16,
+      "! <p class="shorttext">Service Name</p>
+      "! Service name
+      service_name TYPE string,
+    END OF ty_leading_application.
+
+  TYPES:
+    "! <p class="shorttext">Object Type SUCO (Authorization Default Variant)</p>
+    "! Object type SUCO (authorization default variant)
+    BEGIN OF  ty_main,
+      "! $required
+      format_version      TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header              TYPE zif_aff_types_v1=>ty_header_80,
```

Comment:
Is my understanding correct, that you support ABAP language version "default" and "cloudDevelopment" but not "keyUser"? If so, this change should be done to avoid the selection of "keyUser".

```suggestion
      header              TYPE zif_aff_types_v1=>ty_header_80_cloud,
```

If you want to proceed with this, we either need to introduce type `ty_header_80_cloud` or you build your own header. Let me know whether we should extend `zif_aff_types_v1`/`if_aff_types_v1`

Reply:
Yes, I want to proceed with this. Would be great if you extended the interface with type ty_header_80_cloud.

Reply:
@WDFdaniel #669 would be the change to add `zif_aff_types_v1=>ty_header_80_cloud`. Unfortunately, we get errors, because `ty_header_80_cloud` hasn't be used, yet. 

Therefore, I tried to add the change to your PR directly. Unfortunately, I have no permission to perform the changes. Could you provide me access or take over the changes of #669 to your PR?

Reply:
@schneidermic0 Ok, now you should have access to my PR.

Reply:
> @schneidermic0 Ok, now you should have access to my PR.

Thanks, I have add `zif_aff_types_v1=>ty_header_80_cloud` with https://github.com/SAP/abap-file-formats/pull/662/commits/513303f2c90f7288d1385bc09903ad21008e4394

As soon as you use the new type, also the abaplint error should be vanished.